src/vtbackend/Functions.h

```diff
@@ -618,10 +623,82 @@ inline auto const& functions() noexcept
     return funcs;
 }
 
+class SupportedSequences
+{
+  public:
+    SupportedSequences()
```

Comment:
```suggestion
    constexpr SupportedSequences() noexcept
```
in case we cannot avoid this ctor